index.html

```diff
@@ -49,56 +49,56 @@ <h2>Skills</h2>
       </section>
       </div>
 
-      <section id="bio">
+      <section class="details__bio">
```

Comment:
This would also be a new BEM block
```suggestion
      <section class="bio">
```